plugin/methodWrappers.go

```diff
@@ -102,3 +103,11 @@ func IsPrefetched(e Entry) bool {
 func Schema(e Entry) (*EntrySchema, error) {
 	return schema(e)
 }
+
+// List lists the parent's children. It returns a map of <entry_cname> => <entry_object>
+// to optimize querying a specific entry.
+//
+// Note that List's results could be cached.
```

Comment:
"could be" is meant to account for the possibility that someone disabled caching. Open to better ways of wording this because it reads a bit weird.